Add a 'What to ask of Bynder' section to the README (#15)
torchbox/wagtail-bynder: README.md

$ python manage.py update_stale_images --days=3
```

## What to ask of Bynder

When communicating with Bynder about configuring an instance for compatibility with Wagtail, it's important to highlight the derivatives you need them to set up for images and video.

### Why are custom derivatives needed?

> It is common for assets to be uploaded to a DAMS in formats that preserve as much quality as possible. For example, high-resolution uncompressed TIFF images are common for digital photography. These formats are great for print and other media, are simply overkill for most websites. Not only are images likely to be shown at much smaller dimensions in a web browser, but they are also likely to be converted to more web-friendly formats like AVIF or WebP by Wagtail, where the image quality of an uncompressed TIFF is unlikely to shine through.

Author: I decided to simplify this even further to:
> It is common for assets to be uploaded to a DAMS in formats that preserve as much quality as possible. For example, high-resolution uncompressed TIFF images are common for digital photography. While great for print and other media, such formats are simply overkill for most websites. Not only are images likely to be shown at much smaller dimensions in a web browser, but they are also likely to be converted to more web-friendly formats like AVIF or WebP by Wagtail, where the image quality of an uncompressed TIFF is unlikely to shine through.

> You might not notice immediately. But, over time, unnecessarily large source images will have a compounding impact on website performance. Not only must editors wait longer for Wagtail to download the images from Bynder initially, but, every time a new rendition is needed, the original must be loaded into memory from wherever it is stored. Not only will this suck up precious system memory, but the reading of file contents will require longer (often blocking) I/O operations too.

Reviewer: how about:

```suggestion
You might not notice immediately. But, over time, unnecessarily large source images will have a compounding impact on your website performance. Editors will have to wait longer for Wagtail to download the images from Bynder. And every time a new rendition is needed, the original must be loaded into memory from wherever it is stored. This will consume precious system memory, and reading the file contents will require longer (often blocking) I/O operations too.
```

? my brain took forever to parse the "not only ..., but" structures

Author: @zerolab This paragraph now reads:
```
Over time, unnecessarily large source images will have a compounding impact on website performance. Editors will need to wait longer for Wagtail to download the images from Bynder. And, every time a new rendition is needed, the original must be loaded into memory from wherever it is stored, consuming precious system memory, and blocking system I/O operations for longer while reading from storage.
```

#### 'WagtailSource' derivative for images

> What Wagtail really needs is a reliable, high quality derivative, which it can use as a 'source' to generated renditions from. This should strikes the right balance between:

Reviewer: ```suggestion
What Wagtail really needs is a reliable, high quality derivative, which it can use as a 'source' to generate renditions from. This should strike the right balance between:
```

- Retaining as much visual quality as possible, whilst keeping file sizes reasonable. Individual images will naturally vary, but somewhere **between 4MB and 6MB** is a reasonable target.

In most cases, `JPG` will probably the best option. But, for fine art images with lots of uniform colour and sharp edges, `PNG` might be a better fit.

Once configured, URLs for the new derivative should appear in under `"thumbnails"` in the API representation for image assets, like so:

```json
"thumbnails": {
  "mini": "https://orgname.bynder.com/m/3ece125489f192fa/YourGroovyImage.png",
  "thul": "https://orgname.bynder.com/m/3ece125489f192fa/thul-YourGroovyImage.png",
  "webimage": "https://orgname.bynder.com/m/3ece125489f192fa/webimage-YourGroovyImage.png",
  "WagtailSource": "https://orgname.bynder.com/m/3ece125489f192fa/WagtailSource-YourGroovyImage.jpg",
}
```

### 'WebPrimary' and 'WebFallback' derivatives for videos

The goal here is to ensure video can be seen by the widest possibly audience (Wagtail doesn't take a copy of video media like it does for image - as it isn't well equipped for re-encoding it).

> Support for media container formats, video and audio codecs has become more consistant over the years. But, even in 2024, the general consensus is that video on the web should be provided in two different formats in order to work for the widest audience. So, we recommend that Bynder generate **two** custom derivatives for videos:

Reviewer: ```suggestion
Support for media container formats, video and audio codecs has become more consistant over the years. The general consensus is that video on the web should be provided in two different formats in order to work for the widest audience. So, we recommend that Bynder generate **two** custom derivatives for videos:
```
